### ISSUE TITLE
float32 turns into double after conversion

### ISSUE THREAD

**Issue by vladimiroster** [tf-coreml/tf-coreml#242]
Hi,
I've converted my TF model into CoreML. The original data types were float32, but when I'm querying the model with modelDescription, the data type of the multi array is defined as double.
Can this be affecting performance if I'm feeding the model float32? Can I preserve the float32 in the converter?

Thanks!
Vlad

**Comment by aseemw** [comment]
This should not have any significant affect on the performance. Actually for neural networks all the intermediate values use either float32 (on the CPU) or float16 (on the GPU). The input is cast accordingly. 

But still if you want you can change the datatype of the inputs/outputs. See [here](https://github.com/apple/coremltools/blob/691bc991b81b44d211445634780af492083757f8/docs/APIExamples.md#changing-mlmultiarray-inputoutput-datatypes).

**Comment by MarcSteven** [comment]
@aseemw Yeah I want to convert float to double, but it runs and then displayed the error""
